### PR TITLE
Adds error boundary to avoid blank pages on error

### PR DESCRIPTION
**Pull request by josejulio**
I manually added some errors for this gif, but we have seen events when nothing is rendered in the app (only insights chrome is show)
Peek 2020-06-16 14-39

### CODE REVIEW

File: src/pages/Error/Page.tsx [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+    padding: Spacer.SM
+});
+
+const ErrorStack: React.FunctionComponent<ErrorStackProps> = (props) => {
+    const { error } = props;
+
+    if (error.stack) {
+        return (
+            <Text className={ errorClass } component="blockquote">
+                { join(error.stack.split('\n'), 'br') }
```
Comment (spinder) [inline src/pages/Error/Page.tsx:43]: With join, shouldn't 'br', instead be '<br/>' or <br></br>?

**Comment by spinder** [comment]
I couldn't find an easy way to test this.  I only have one question which I inserted as an inline comment, but this looks good from what I see. We're just replacing the Router with a component that includes that same Router and some extra messaging/display.  Ready to merge to master if the comment in the issue causes no changes.

File: src/pages/Error/Page.tsx (hunk 1 above)
Comment (josejulio) [reply src/pages/Error/Page.tsx:43]: 'br' is fine. 
The argument is a component type (or name), when is a string, a raw html tag is used. 

Inside the code the passed component is used as `<TheComponent/>`.

**Comment by josejulio** [comment]
Using an error boundary: https://reactjs.org/docs/error-boundaries.html

File: src/properties/Messages.ts [hunk 2]
```diff
@@ -16,6 +16,17 @@ const MutableMessages = {
                 content: 'Contact your organization administrator(s) for more information.'
             }
         },
+        error: {
+            title: 'Policies',
+            emptyState: {
+                title: 'Unhandled error',
+                content: 'There was a problem trying to process you request.',
```
Comment (katierik) [inline src/properties/Messages.ts:23]: Typo here: Should be "your"

**Comment by katierik** [comment]
UXD has not defined an error state for this. So far, this looks good but I may follow up with some changes to the UI. (content changes, or the show details change, the overall layout is good)

File: src/properties/Messages.ts (hunk 2 above)
Comment (josejulio) [inline src/properties/Messages.ts:23]:
```suggestion
                content: 'There was a problem trying to process your request.',
```

Comment (josejulio) [reply src/properties/Messages.ts]: Thanks!

File: src/pages/Error/Page.tsx (hunk 1 above)
Comment (spinder) [reply src/pages/Error/Page.tsx:43]: Thx. I need to go read up on  https://reactjs.org/docs/error-boundaries.html some more. Merging this to master.

**Comment by spinder** [comment]
Merging after suggested change. Remaining failure is due to api change. Will be addressed in another pr.

**Pull request merged**